docs/writing-tests/reftests.md

```diff
@@ -3,7 +3,9 @@
 Reftests are one of the primary tools for testing things relating to
 rendering; they are made up of the test and one or more other pages
 ("references") with assertions as to whether they render identically
-or not.
+or not. This page describes their aspects exhaustively; [the tutorial
+on writing a reftest](reftest-tutorial) offers a more limited but
```

Comment:
```suggestion
on writing a reftest](reftest-tutorial) offers a more-limited but
```

Reply:
I'm clearly not accustomed to hyphenating compound adjectives, but it seems to me that this hyphen is superfluous due to the word "a". For instance, the phrases "offers more limited choices" and "offers more-limited choices" mean
two different things, but the phrases "offers a more limited set of choices" and "offers a more-limited set of choices" do not.

Reply:
Hyphenating compound adjectives is basically never strictly necessary. I agree it would be more warranted if the word "a" weren’t there. But it’s never harmful to hyphenate a compound adjective. It certainly doesn’t ever make the meaning less ambiguous. It just helps for the case where somebody might otherwise trip over when reading. But I don’t feel strongly about it. Judgment call (yours — since you’re the one who did the work to write the actual content).